applications/CoSimulationApplication/python_scripts/coupling_operations/nodal_data_to_elemental_data.py

```diff
@@ -0,0 +1,64 @@
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_coupling_operation import CoSimulationCouplingOperation
+from KratosMultiphysics.CoSimulationApplication import ConversionUtilities
+
+# CoSimulation imports
+import KratosMultiphysics.CoSimulationApplication.co_simulation_tools as cs_tools
+
+def Create(*args):
+    return NodalToElementalData(*args)
+
+class NodalToElementalData(CoSimulationCouplingOperation):
+    """This operation maps the Nodal Data to Elemental Data for a given ModelPart consistently. The elemental value is an averaged value of Nodal values.
+    """
+    def __init__(self, settings, solver_wrappers, process_info, data_communicator):
+        super().__init__(settings, process_info, data_communicator)
+        solver_name = self.settings["solver"].GetString()
+        data_name = self.settings["data_name"].GetString()
+        self.interface_data = solver_wrappers[solver_name].GetInterfaceData(data_name)
+        self.variable = self.interface_data.variable
+        self.dimension = self.interface_data.dimension
+        self.consistent = self.settings["consistent"].GetBool()
+
+        if not self.consistent:
+            raise RuntimeError("Conservative mapper from nodes on elements is not implemented!")
```

Comment:
why adding this if it anyway only throws?

Reply:
I am a fan of only adding options when they are actually used

Besides, I really dislike the consistent/conservative terminology bcs its strongly tied to FSI, which CoSim is not
I use "distribute_values" in many places for this purpose (or "use_transpose" for the mapper)